app/experimenter/nimbus-ui/src/components/FormAudience/index.test.tsx

```diff
@@ -52,18 +53,19 @@ describe("FormAudience", () => {
     const submitErrors = {
       "*": ["Big bad server thing happened"],
       channel: ["Cannot tune in this channel"],
-      firefoxMinVersion: ["Bad min version"],
-      targetingConfigSlug: ["This slug is icky"],
-      populationPercent: ["This is not a percentage"],
-      totalEnrolledClients: ["Need a number here, bud."],
-      proposedEnrollment: ["Emoji are not numbers"],
-      proposedDuration: ["No negative numbers"],
+      firefox_min_version: ["Bad min version"],
+      targeting_config_slug: ["This slug is icky"],
+      population_percent: ["This is not a percentage"],
+      total_enrolled_clients: ["Need a number here, bud."],
+      proposed_enrollment: ["Emoji are not numbers"],
+      proposed_duration: ["No negative numbers"],
     };
     const { container } = render(<Subject submitErrors={submitErrors} />);
     await waitFor(() => {
       expect(screen.queryByTestId("FormAudience")).toBeInTheDocument();
     });
-    for (const [fieldName, [error]] of Object.entries(submitErrors)) {
+    for (const [submitErrorName, [error]] of Object.entries(submitErrors)) {
+      const fieldName = snakeToCamelCase(submitErrorName);
```

Comment:
I wonder if there's merit in making everything snake case across the board 🤔

Reply:
Yeah, would be nice if server & client were consistent, but Python & JavaScript have different case traditions and I have run into weird issues where some magic on the Python/Django side assumes snake-case.

Had also considered wrapping submitErrors such that any mutations did the snake-to-camel conversion up front. What I ended up doing feels cumbersome but maybe more straightforward.